event/listener.php

```diff
@@ -78,7 +78,6 @@ static public function getSubscribedEvents()
 
 			// Message parsing
 			'core.modify_text_for_display_before'			=> 'replace_mentions_for_display',
-			'core.modify_text_for_display_before'			=> 'replace_mentions_for_display',
```

Comment:
@s9e Oh, this is the same event indeed, ye. But I was quite sure I need another one here.
Should be `core.modify_text_for_display_before`, shouldn't it?
I think I've seen bot events in use by advanced bbcode extension.

When this part is clear, I'll merge that pull request.